__main__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
```

Comment:
Didn't the agent step up to requiring Python 2.7?

Reply:
Not sure about that... same comment in all source files (__main__.py was an existing file) and Travis tests 2.6 (and 2.7).

Reply:
I ran into some issues with Python 2.6 and disabled it in my CI update PR.  At this point 2.6 probably isn't worth maintaining anyway.  Note every python file has a comment about 2.6 in it, which should be removed.